.github/workflows/ci-uxd.yml

```diff
@@ -0,0 +1,144 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  
+permissions:
+  contents: read
+  pull-requests: write
+  
+env:
+  solana_version: 1.8.11
+  anchor_version: 0.20.0
+  cluster: "devnet"
+  runner_wallet_path: "/home/runner/.config/solana/id.json"
+  mango_account_mm: "8fbL4156uoVYYyY9cvA6hVBBTdui9356tdKmFbkC6t6w"
+  NODE_AUTH_TOKEN: ${{ secrets.NODE_AUTH_TOKEN }}
```

Comment:
There are two secrets that will need to be added to the mango repository if that's fine? In order to access to our private NPM package on gh registry and to checkout the private repo